tests/data/taca_test_cfg.yaml

```diff
@@ -0,0 +1,89 @@
+log:
+    log_file: data/taca.log
+
+storage:
+    data_dirs:
+        - /srv/illumina/hiseq_data
+        - /srv/illumina/miseq_data
+    archive_dirs:
+        - /srv/illumina/hiseq_data/nosync
+        - /srv/illumina/miseq_data/nosync
+    irods:
+        irodsHome: /ssUppnexZone/proj/a2010002
+
+cleanup:
```

Comment:
@senthil10 @b97pla are this properties (cleanup) necessary for the tests? as they contain real usernames and project names I will just remove them if they're not necessary